### DISCUSSION TITLE
RaspAP and Nordvpn Meshnet

### DISCUSSION THREAD

**Discussion by djtp** [RaspAP/raspap-webgui#1577] [Q&A]
Hello - my first post, as I've found most of my answers previous, so sorry if this is a noob question.
I had a grand idea of being able to connect one Raspberry Pi (Remote) to another Raspberry Pi (Home) in a different country, using Nord VPN's meshnet feature.  I do a lot of travelling and want to take the Remote with me (connected via ethernet and generating a hotspot) to always be able to connect back to Home

This works! However, I then attempted to create a wifi hotspot to connect, using RaspAP. I cannot seem to figure out how to connect RaspAP to the Meshnet from NordVPN. Can this be done?  Is it even possible?

In summary - meshnets connect, I can connect to RaspAp - but I cannot get the RaspAp connection to flow through to the meshnet tunnel.  Any ideas and help are gratefully received!
Am I over-complicating things (again)?

**Comment by geoff5283** [comment]
I'm not too sure if I fully understand your scenario but if you're using the first-party app when you first set up your RaspAP and meshnet is working; You might want to make sure LAN discovery is on.

`nordvpn set lan-discovery enable`

**Comment by pravinvekaria** [comment]
> Hello - my first post, as I've found most of my answers previous, so sorry if this is a noob question. I had a grand idea of being able to connect one Raspberry Pi (Remote) to another Raspberry Pi (Home) in a different country, using Nord VPN's meshnet feature. I do a lot of travelling and want to take the Remote with me (connected via ethernet and generating a hotspot) to always be able to connect back to Home
> 
> This works! However, I then attempted to create a wifi hotspot to connect, using RaspAP. I cannot seem to figure out how to connect RaspAP to the Meshnet from NordVPN. Can this be done? Is it even possible?
> 
> In summary - meshnets connect, I can connect to RaspAp - but I cannot get the RaspAp connection to flow through to the meshnet tunnel. Any ideas and help are gratefully received! Am I over-complicating things (again)?

@djtp I am looking to do exactly this, did you manage to get it working?

**Reply by djtp** [reply]
Unfortunately not - I have tried a few more things, but other projects took over - still searching however!
